[docs-only] Add readme.md note to not use xattrs anymore (#6224)
owncloud/ocis: services/storage-system/README.md

Purpose and description to be added

## Deprecated Metadata Backend

> Starting with ocis version 3.0.0, the default backend for metadata switched to messagepack. If the setting `STORAGE_SYSTEM_OCIS_METADATA_BACKEND` has not been defined manually, the backend will be migrated to `messagepack` automatically. Though still possible to manually configure `xattrs`, this setting should not be used anymore as it will be removed in a later version.

Reviewer: ```suggestion
Starting with Infinite Scale version 3.0.0, the default backend for metadata switched to messagepack. If the setting `STORAGE_SYSTEM_OCIS_METADATA_BACKEND` has not been defined manually, the backend will be migrated to `messagepack` automatically. Though still possible to manually configure `xattrs`, this setting should not be used anymore as it will be removed in a later version.
```

> Starting with ocis version 3.0.0, the default backend for metadata switched to messagepack. If the settings `STORAGE_SYSTEM_OCIS_METADATA_BACKEND` **AND** `STORAGE_USERS_OCIS_METADATA_BACKEND` have not been defined manually, the backend will be migrated to `messagepack` automatically. Though still possible to manually configure `xattrs`, this setting should not be used anymore as it will be removed in a later version.

Author: ```suggestion
Starting with ocis version 3.0.0, the default backend for metadata switched to messagepack. If the setting `STORAGE_SYSTEM_OCIS_METADATA_BACKEND` has not been defined manually, the backend will be migrated to `messagepack` automatically. Though still possible to manually configure `xattrs`, this setting should not be used anymore as it will be removed in a later version.
```
